docs/src/index.md

```diff
@@ -0,0 +1,41 @@
+# ImageDraw.jl Documentation
+
+A drawing package for JuliaImages
+
+```@contents
+Depth = 3
+```
+
+## introduction
+
+ImageDraw supports basic drawing on Images. You can draw points, lines, circles, ellipse and paths.  
+
+
+## Basic usage
+
+Let's start with a drawing a circle
+
+```julia
+julia> using TestImages, ImageDraw
+
+julia> img = testimage("lighthouse");
+
+julia> draw(copy(img), Ellipse(CirclePointRadius(350,200,100)));
+
+```
+When displayed, these three images look like this:
+
+![Draw Circle](images/lighthouse_circle.png)
```

Comment:
The assets can be generated using `@example` block so that we don't need to *frequently* check-in these images, see [ImageEdgeDetections Adds documentation and examples](https://github.com/JuliaImages/ImageEdgeDetection.jl/pull/6/files#diff-493d89206c11fea99259d0771034d26aR56-R70) for an example.

Reply:
Done